File: Hipotenusa/Punto.vb
Public Class Punto
    Public Xpunto1 As Double
    Public Ypunto1 As Double
    Public Xpunto2 As Double
    Public Ypunto2 As Double

    Public Property Xpropiedad1() As String
        Get
            Return Xpunto1
        End Get
        Set(value As String)
            Xpunto1 = value
        End Set
    End Property

    Public Property Ypropiedad1() As String
        Get
            Return Ypunto1
        End Get
        Set(value As String)
            Ypunto1 = value
        End Set
    End Property

    Public Property Xpropiedad2() As String
        Get
            Return Xpunto2
        End Get
        Set(value As String)
            Xpunto2 = value
        End Set
    End Property

    Public Property Ypropiedad2() As String
        Get
            Return Ypunto2
        End Get
        Set(value As String)
            Ypunto2 = value
        End Set
    End Property

    Public Sub Puntos(ByVal X1 As Double, ByVal Y1 As Double, ByVal X2 As Double, ByVal Y2 As Double)
        Xpunto1 = X1
        Ypunto1 = Y1
        Xpunto2 = X2
        Ypunto2 = Y2
    End Sub


    Public Function Operacion(X1, Y1, X2, Y2)
        Dim distancia As Double
        distancia = Math.Sqrt((Math.Pow((X2 - X1), 2)) + (Math.Pow((Y2 - Y1), 2)))
        Return distancia
    End Function
End Class

File: Hipotenusa C/Punto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hipotenusa_C
{
    public class Punto
    {

        private double Xpunto1;
        private double Ypunto1;
        private double Xpunto2;
        private double Ypunto2;

        public double Xpropiedad1 { get => Xpunto1; set => Xpunto1 = value; }
        public double Ypropiedad1 { get => Ypunto1; set => Ypunto1 = value; }
        public double Xpropiedad2 { get => Xpunto2; set => Xpunto2 = value; }
        public double Ypropiedad2 { get => Ypunto2; set => Ypunto2 = value; }

        public void Puntos(double X1, double Y1, double X2, double Y2) {
            Xpunto1 = X1;
            Ypunto1 = Y1;
            Xpunto2 = X2;
            Ypunto2 = Y2;
   }


        public double Operacion (double X1, double Y1, double X2, double Y2) {
            double distancia;
            distancia = Math.Sqrt((Math.Pow((X2 - X1), 2)) + (Math.Pow((Y2 - Y1), 2)));
                return distancia;
         }
    }
}
